File: app/src/main/java/com/messi/languagehelper/box/CollectedData.java
package com.messi.languagehelper.box;


import io.objectbox.annotation.Entity;
import io.objectbox.annotation.Id;
import io.objectbox.annotation.Index;

@Entity
public class CollectedData {

    @Id
    private long id;
    @Index
    private String objectId;
    private String name;
    private String json;
    private String type;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getObjectId() {
        return objectId;
    }

    public void setObjectId(String objectId) {
        this.objectId = objectId;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getJson() {
        return json;
    }

    public void setJson(String json) {
        this.json = json;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }
}

File: app/src/main/java/com/messi/languagehelper/box/CollectedData.kt
package com.messi.languagehelper.box

import io.objectbox.annotation.Entity
import io.objectbox.annotation.Id
import io.objectbox.annotation.Index

@Entity
data class CollectedData (
    @Id var id: Long = 0,
    @Index var objectId: String? = null,
    var name: String? = null,
    var json: String? = null,
    var type: String? = null
)